standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -10049,9 +10053,6 @@ public boolean deletePartitionColumnStatistics(String catName, String dbName, St
         }
       }
       ret = commitTransaction();
-    } catch (NoSuchObjectException e) {
-      rollbackTransaction();
-      throw e;
```

Comment:
Hey, how is this related to closing query objects?

Reply:
It's not related to the query closing, here just to cleanup some unnecessary codes.